* program
** Program
*** aDNA
*** genetic drift
*** exercise 1
*** How this is useful for the past
*** Horse domestication and ancient DNA
*** exercise 2
** objective
   The main objective of this Lecture + Exercise (LE) is to show how genetic drift can be used to investigate the genetic relationship of the past. Specifically, we will use two methods to reconstruct the genetic relationship of the horse back through time. Second objective of this LE is to become familiar with two often used methods in population genomics studies, namely f3-statistics and treemix, and learn how to interpret these to understand the evolutionary history. Lastly, we will reconstruct some of results used in a recent publication on the ancestors of domesticated horse we know today.
* lecture
** Introduce myself ( 10 min )
*** masters in bioinformatics
*** Phd in Ancient DNA at geogenetics, Denmark and Toulouse, France
**** population genomics of horses
**** methylation signatures in ancient DNA seq data
** introduce aDNA (use masters intro or from DTU teaching) (20 min)
** recap genetic drift from book and previous lecture/exercise (10 min)
   [[https://www.apsnet.org/edcenter/advanced/topics/PopGenetics/Pages/geneticdrift.aspx][simple drift explanation]]
** exercise 1 (driftr) (15 min)
   [[https://github.com/FerRacimo/KUPopGenMasterCourse/blob/master/WrightFisherTutorial.md][use this as inspiration for the drift analyses]]
*** install
    Use the following code to install driftR locally
    #+BEGIN_SRC bash
      git clone https://github.com/cjbattey/driftR.git
      cd driftR
    #+END_SRC

    #+BEGIN_SRC R
      ## https://gist.github.com/stevenworthington/3178163
      ipak <- function(pkg){
      new.pkg <- pkg[!(pkg %in% installed.packages()[, "Package"])]
      if (length(new.pkg))
          install.packages(new.pkg, dependencies = TRUE)
      sapply(pkg, require, character.only = TRUE)
      }

      ipak(c("shiny", "viridis", "reshape", "ggplot2", "plyr", "magrittr"))
      pathTodriftR <- "/home/krishang/devel/driftR"
      runApp(appDir=pathTodriftR)
    #+END_SRC
*** Exercise text
    In this exercise, we will explore the effect of genetic drift on allele frequencies using simulations.

    This software has a series of parameters to tune. Here, we will explore how the population size impacts the genetic drift. The following parameters will therefore be fixed:
     - Starting allele frequency: 0.5
     - Mutation rate: 0
     - Fitness of genotype AA, AB, and BB: 1
     - Migration rate: 0
     - Number of populations: 10
     - Number of generations: 100
*** Questions
    - Press 'Run Simulation' and describe what you see. What is the on the axis and what does every line show?
    - What happens if we set the populations size to 1000
    - What happens if we set the populations size to 10000
    - Run 20 replicates of the last three runs (population size: 100, 1000, 10000)
      - Does the mean allele frequency (p) change?
      - Does the variance allele frequency (p) change?
    - Are these observations in line with your expectations?
    - Would you expect to observe a similar behavior of the mean and variance of the allele frequency if we reduce the fitness of genotype AB and BB to 0.9? If not, what would change?
    - What are the underlying mechanisms of these observations?
*** Summarize the results
** TODO What genetic drift can say about the genetic relationships in the past (10 min)
** TODO Introduction to horse domestication and ancient DNA (20 min)
*** Why do we even care about horse domestication
*** Why ancient DNA is a fundamental tool investigate horse domestication
*** Botai horses and the archaeological context (when, where, evindence for domestication)
*** What samples we have used to analyzed this
**** draw some trees showing the hypothesis
** TODO exercise 2 (f3 and treemix on horses) ( total 10+20+20+20 = 70 min)
*** Introduction to exercise two (10 min)
    In exercise 2 we investigate if the botai horses are the ancestors of present day domesticated horses. To unravel this we will use two different methods, namely f3-outgroup and treemix. It is common to both methods that they are based on genetic drift and to what extent pairs of horses/populations shared genetic drift in the past.
*** F3 (20 min)
    The first method we will apply to the method is named f3-outgroup. This method uses an outgroup (Donkey) to recover the amount of shared genetic drift between two horses/populations. The statistic measures the branch length from the outgroup and split of the two populations. It is defined as the product of allele frequency differences between C-A and C-B.

    !! make a figure of f3 and outgroup-f3 to see the difference

    You will compute all pairwise permutations of the horses analyzed in this study and plot it as a heatmap. These results can give us the first indication of the genetic placement of the first domesticated horses.
**** Questions 
     - Why are the f3-outgroup values so high for Przewalski horses.
     - What are the main groups of horses that share more within populations genetic drift
     - What Horses share the most drift with Botai horses?
     - Does these results indicate that Botai horses are the direct ancestors of the domesticated horses we know today?
       - If not what does the results then indicate?
*** Treemix (20 min)
    The second method is named treemix and is used to reconstruct the genetic relationship between populations based on genetic drift. In this part you will generate an evolutionary tree using treemix of the same subset of horses as used in the previous analyses with the exception of the outgroup. Instead of using the donkey as outgroup, we use xxx,xxx,xxx. These three samples represent a horse lineage that went extinct 5k years ago. 
    Vertical lines do not have a biological meaning. 
**** Questions
     - Define the two monophyletic groups in addition two the outgroup.
     - Is the placement of Botai horses expected if they are the ancestors of the present day domesticated horses.
     - What populations shares the most genetic drift the botai horses?
     - Is there a difference between the length of the terminal branches in the two monophyletic groups?
       - If yes, explain why they differ?
*** Summarize (20min)
    Go through the results and discuss them. Shown the format statistical test of direct ancestry with the trees from the PhD. 
* what to read
** nielsen and slatkin p. 21-26
** treemix - paper [[https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002967][link]]

** gaunitz et al [[http://science.sciencemag.org/content/360/6384/111.abstract][link]]
